examples/README.md

```diff
@@ -1,69 +1,72 @@
-## Prerequisites:
+# RHOAS GO SDK examples
 
-- RHOAS CLI installed - it will be used for obtaining tokens for authentication.
-For more information about authentication please refer to SDK docs.
-- Repository is properly initiatilized: 
-
-```shell
-go mod tidy
-```
+## Prerequisites
 
+1. Go to  <https://console.redhat.com/openshift/token> and press "Load token".
+2. Copy token fro clipboard to file
+3. Set env variable containing token `OFFLINE_TOKEN`
 
 ## Running examples
 
 To run examples execute go run, using ACCESS_TOKEN along with API_URL where necessary.
 
-### For example:
+### For example
 
-### For kafkamgmt: 
+### For kafkamgmt
 
 ```shell
-ACCESS_TOKEN=`rhoas authtoken` go run ./examples/kafkamgmt/kafka_mgmt.go
+go run ./examples/kafkamgmt/kafka_mgmt.go
```

Comment:
should we be using 
```bash
OFFLINE_TOKEN=`ocm token` go run ./examples/kafkamgmt/kafka_mgmt.go
```
here, this is the same token but expires quicker? unless I have this wrong (which I may do 😆 )

Reply:
This token never expires